extensions/functions_aggregate_generic.yaml

```
@@ -16,7 +17,8 @@ aggregate_functions:
     description: "Count a set of records (not field referenced)"
     impls:
       - args:
-          - options: [SILENT, SATURATE, ERROR]
+          - name: x
```

In this case, it should actually be named `overflow`.

```suggestion
          - name: overflow
```